libs/service-portal/graphql/src/lib/client.ts

```diff
@@ -9,8 +9,13 @@ import { onError } from '@apollo/client/link/error'
 import { RetryLink } from '@apollo/client/link/retry'
 import { setContext } from '@apollo/client/link/context'
 
+const apiUri =
```

Comment:
I think this would be described better as a runtime config instead of doing a location check

Reply:
We could do that